README.md

```diff
@@ -25,12 +25,14 @@ Currently the following variables are supported:
 * `ocp_install_url` - Required, http(s) URL to the OCP Installer archive. A recipe for
   setting this automatically is included in the examples below.
 * `ocp_install_path` - Default: `/usr/local/bin/`. Destination directory for installed
-   `openshift-installer` binary. This location should be in the system `PATH`.
```

Comment:
Since this role also happens to be what puts the installer on the target system, the target location doesn't actually need to be on the `PATH`; the role knows where it is and calls it explicitly now.